_layouts/default.html

```diff
@@ -24,7 +23,7 @@
   <body class="grey darken-4">
     {% include navigation.html %}
     {{ content }}
-
+  
```

Comment:
Revert the addition of unwanted whitespace.

```suggestion

```

Reply:
lol